core/psh/pshapp/auth.c

```diff
@@ -111,35 +111,35 @@ static int psh_auth(int argc, char **argv)
 	if (isatty(STDOUT_FILENO) == 0) {
 		sleep(1);
 		fprintf(stderr, "psh: unable to login, not a tty\n");
-		return -ENOTTY;
+		return ENOTTY;
```

Comment:
note for the future: i will go for returning 1 instead of errno here (see `man ping`, `man curl` and others - exit codes does not correspond to errno values). To be done in separate PR.